js/breadcrumb.js.php

```diff
@@ -11,7 +11,9 @@
 	if (strpos($referer, 'optioncss=print') !== false ) {
 		exit;
 	}
-
+	
+	$publicpage = strpos($referer, 'public');
```

Comment:
To limit false positive i suggest :
 
```suggestion
	$publicpage = strpos($referer, '/public/');
```

Reply:
ok